src/main/kotlin/stringaddcalculator/domain/DelimiterFinder.kt

```diff
@@ -0,0 +1,23 @@
+package stringaddcalculator.domain
+
+object DelimiterFinder {
+    private val BASIC_DELIMITERS = listOf(",", ":")
+    private val CUSTOM_DELIMITER_REGEX = Regex("//(.)\n(.*)")
+
+    fun find(string: String): List<String> {
+        if (string.hasCustomDelimiter()) {
+            println(findCustomDelimiter(string))
+            return BASIC_DELIMITERS.plus(findCustomDelimiter(string))
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            return BASIC_DELIMITERS + findCustomDelimiter(string)
```